src/os/time/time.js

```diff
@@ -896,3 +901,36 @@ os.time.combineDateTime = function(days, time, opt_daysFormat = 'YYYY-MM-DD', op
   var combinedTime = os.time.parseMoment(`${days} ${time}`, `${opt_daysFormat} ${opt_timeFormat}`, true);
   return combinedTime;
 };
+
+
+/**
+ * Determines whether a duration is relative or not
+ *
+ * @param {string} duration The duration that will be analyzed
+ * @return {boolean} Whether the duration is relative or not
+ */
+os.time.isRelativeDuration = function(duration) {
+  var isRelativeDuration = false;
```

Comment:
I'm wondering if we should make an assumption about the duration string to simplify this function and future maintenance. If we assume relative durations will always be "last xx", we can make this function dead simple:

```
/**
 * Regular expression to test for a relative duration string.
 * @type {RegExp}
 */
os.time.RELATIVE_DURATION_REGEXP = /^last /i;

os.time.isRelativeDuration = function(duration) {
  return os.time.RELATIVE_DURATION_REGEXP.test(duration);
};
```

Then if the relative options change in the future, we won't have to update this function assuming they follow the same naming scheme.